src/wallet.rs

```diff
@@ -514,7 +516,11 @@ impl Wallet {
         Ok(sidechains)
     }
 
-    pub async fn get_ctip(
+    pub async fn get_mainchain_tip(&self) -> Result<bitcoin::BlockHash> {
```

Comment:
Is this actually async? I don't see an `await`

Reply:
You're right! Don't know how that got in there. Changing.